packages/govuk-frontend/src/govuk/helpers/_typography.scss

```
@@ -62,6 +62,23 @@
   font-weight: $govuk-font-weight-bold if($important, !important, null);
 }
 
+/// Tabular number helper
+///
+/// Switches numerical glyphs (0–9) to use alternate forms with a
```

Sorry me again with my British words 🎩

```suggestion
/// Switches numerical glyphs (0–9) to use alternative forms with a
```

---

Pip pip ☕